packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -222,6 +228,19 @@ export class Autosuggest {
     });
   }
 
+  private markAutosuggestMarkItems(items: AutosuggestMarkItem[]): (VNode | string)[] {
```

Comment:
Lokale markeer (`private markTerms()`) functie uitkleden en de vertaalslag naar JSX in eigen functie plaatsen. De output van de lokale en externe markeer functie wordt door deze nieuwe functie gebruikt.

De `private markTerms()` moet als return value ook `AutosuggestMarkItem[]` zijn.

Reply:
Ik loop hier vast op de returntype en de recursieve aanroep van `makeTerms`. Kunnen we hier even samen naar kijken?

Reply:
Zeker!

Reply:
Gedaan. En bedankt.